src/controllers/dynakube/updates/version.go

```diff
@@ -36,7 +36,15 @@ func ReconcileVersions(
 		!dk.FeatureDisableActiveGateUpdates() &&
 		dkState.IsOutdated(dk.Status.ActiveGate.LastUpdateProbeTimestamp, ProbeThreshold)
 
-	if !needsActiveGateUpdate && !needsOneAgentUpdate {
+	needsEecUpdate := dk.NeedsActiveGate() &&
```

Comment:
same here, these images shouldn't be updated when they are not required

Reply:
Makes sense